learningresources/tests/test_api.py

```diff
@@ -157,36 +152,6 @@ def test_invalid_create_repo(self):
         self.assertIn("description", ex.exception.args[0])
 
 
-class TestStaticAsset(LoreTestCase):
-    """
-    Verify the Python API for static assets.
-    """
-    def test_create_static_asset(self):
-        """
-        Validate that we can create assets.
-        """
-        basename = 'blah.txt'
-        file_contents = b'hello\n'
-        with tempfile.TemporaryFile() as temp:
-            temp.write(file_contents)
-            test_file = File(temp, name=basename)
-            asset = create_static_asset(self.course.id, test_file)
-            self.addCleanup(default_storage.delete, asset.asset)
-            self.assertFalse(test_file.closed)
-            self.assertEqual(asset.course, self.course)
-            self.assertEqual(
-                static_asset_basepath(asset, basename),
-                asset.asset.name
-            )
-
-            # Close file and assert that exception is raised.
-            test_file.close()
-            test_file.name = 'new_name'
-            with self.assertRaises(ValueError):
-                asset = create_static_asset(self.course.id, test_file)
-        self.assertEqual(file_contents, asset.asset.read())
-
-
```

Comment:
There aren't any unit tests for `create_static_assets` that I can find, just the big integration test.  We should have both